cmake/winml.cmake

```diff
@@ -81,7 +81,7 @@ target_cppwinrt(winml_api
   ${sdk_folder}              # location of sdk folder
   ${sdk_version}             # sdk version
   ${target_folder}           # the folder this target will be placed under
-  ${winml_midl_defines}      # the midl compiler defines
+  "${winml_midl_defines}"    # the midl compiler defines
```

Comment:
This is unrelated to the test break. It fixes project generation when `${winml_midl_defines}` is not defined and target_cppwinrt would be called with less than its minimum number of arguments. This regression came with the WinML namespace changes.